src/pg/sql/10_moran.sql

```diff
@@ -10,28 +10,101 @@ CREATE OR REPLACE FUNCTION
       num_ngbrs INT DEFAULT 5)
 RETURNS TABLE (moran NUMERIC, significance NUMERIC)
 AS $$
+  plpy.execute('SELECT cdb_crankshaft._cdb_crankshaft_activate_py()')
   from crankshaft.clustering import moran_local
   # TODO: use named parameters or a dictionary
   return moran(subquery, attr, num_ngbrs, permutations, geom_col, id_col, w_type)
 $$ LANGUAGE plpythonu;
 
 -- Moran's I Local
 CREATE OR REPLACE FUNCTION
-  CDB_AreasOfInterest_Local(
+  _CDB_AreasOfInterest_Local(
       subquery TEXT,
       attr TEXT,
-      permutations INT DEFAULT 99,
-      geom_col TEXT DEFAULT 'the_geom',
-      id_col TEXT DEFAULT 'cartodb_id',
-      w_type TEXT DEFAULT 'knn',
-      num_ngbrs INT DEFAULT 5)
+      permutations INT,
+      geom_col TEXT,
+      id_col TEXT,
+      w_type TEXT,
+      num_ngbrs INT)
 RETURNS TABLE (moran NUMERIC, quads TEXT, significance NUMERIC, ids INT, y NUMERIC)
 AS $$
+  plpy.execute('SELECT cdb_crankshaft._cdb_crankshaft_activate_py()')
   from crankshaft.clustering import moran_local
   # TODO: use named parameters or a dictionary
   return moran_local(subquery, attr, permutations, geom_col, id_col, w_type, num_ngbrs)
 $$ LANGUAGE plpythonu;
 
+CREATE OR REPLACE FUNCTION
+  CDB_AreasOfInterest_Local(
+    subquery TEXT,
+    attr TEXT,
+    permutations INT DEFAULT 99,
+    geom_col TEXT DEFAULT 'the_geom',
+    id_col TEXT DEFAULT 'cartodb_id',
+    w_type TEXT DEFAULT 'knn',
+    num_ngbrs INT DEFAULT 5)
+RETURNS TABLE (moran NUMERIC, quads TEXT, significance NUMERIC, ids INT, y NUMERIC)
+AS $$
+
+  SELECT moran, quads, significance, ids, y
+  FROM cdb_crankshaft._CDB_AreasOfInterest_Local(subquery, attr, permutations, geom_col, id_col, w_type, num_ngbrs);
+
+$$ LANGUAGE SQL;
+
+CREATE OR REPLACE FUNCTION
+  CDB_GetSpatialHotspots(
```

Comment:
Great -- we would prefer to stick with the CamelCase convention that is currently used in Data Observatory. Since we will be maintaining both of these code bases it will be easier for us. I will update the functions in this pull request to reflect that, and do a separate PR that moves all of the others over. @stuartlynn and I talked about this already so he's aware.